analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -681,7 +681,8 @@ def skip_cpp(compile_actions, skip_handler):
 def start_workers(actions_map, actions, context, analyzer_config_map,
                   jobs, output_path, skip_handler, metadata,
                   quiet_analyze, capture_analysis_output, timeout,
-                  ctu_reanalyze_on_failure, statistics_data, manager):
+                  ctu_reanalyze_on_failure, statistics_data, manager,
+                  cmp_cmd_count):
```

Comment:
For me the `cmp` abbreviation means `compare`, so `cmp_cmd_count` means for me `compare command count`. It is a little bit misleading for me. Can we use `compile_cmd_count` as a variable name instead of this? Use this in other places too if you agree with this.